l10n_it_ricevute_bancarie/migrations/12.0.1.6.1/post-migrate.py

```diff
@@ -0,0 +1,10 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    invoices = env['account.invoice'].search([
+        ('payment_term_id.riba', '=', True), ('riba_partner_bank_id', '=', False)
+    ])
+    for invoice in invoices:
+        invoice._onchange_riba_partner_bank_id()
```

Comment:
questo onchange scrive nel DB senza bisogno di `_convert_to_write`?

Reply:
Nei test ha funzionato senza, meglio aggiungerlo?

Reply:
Se funziona va bene, ma mi chiedevo come fa

Reply:
In realtà me lo sono chiesto anche io 😬 possibile l'utilizzo di delle librerie `openupgradelib`?

Reply:
Ecco: https://discord.com/channels/753902328494424064/753902328494424070/807184531554828329